canvasapi/course.py

```diff
@@ -541,12 +544,33 @@ def get_external_tools(self, **kwargs):
             _kwargs=combine_kwargs(**kwargs)
         )
 
-    def get_section(self, section):
+    def get_sections(self, **kwargs):
+        """
+        List all sections in a course.
+
+        :calls: `GET /api/v1/courses/:course_id/sections \
+        <https://canvas.instructure.com/doc/api/sections.html#method.sections.index>`_
+
+        :rtype: :class:`canvasapi.paginated_list.PaginatedList` of
+            :class:`canvasapi.section.Section`
+        """
+        from canvasapi.section import Section
+
+        return PaginatedList(
+            Section,
+            self._requester,
+            'GET',
+            'courses/{}/sections'.format(self.id),
+            {'course_id': self.id},
```

Comment:
I don't believe it is necessary to manually include `course_id` as Canvas' response already includes `course_id`.

Reply:
amended